.github/workflows/cd.yml

```diff
@@ -0,0 +1,72 @@
+name: merge-build
+on:
+  push:
+    branches:
+      - master
+      - cloud
+      - release-*
+defaults:
+  run:
+    shell: bash
```

Comment:
This can be omitted since bash is the default shell for Github Actions running on linux.